Automattic/pocket-casts-android
# Add What's New for Reimagine Sharing

## Description

See for specs: p1724799780302989-slack-C02LGFKP0AE

## Testing Instructions

1. Install the app from 028c4652f9f71d1b4d28bd75722341f6e7d93f72
2. Open the app.
3. Make sure that you close the onboarding screen.
4. Install the app from this branch.
5. Open the app.
6. You should see "What's New"

## Screenshots or Screencast 

Screenshots do not account for [1e382c9](https://github.com/Automattic/pocket-casts-android/pull/2745/commits/1e382c972cfdd41676c3903f98661d90c8668344)

| Animation | Small device | Fold device |
| - | - | - |
| ![anim](https://github.com/user-attachments/assets/2fa56164-7730-417e-a15a-a5ce7a6a77fb) | ![small](https://github.com/user-attachments/assets/67cc9f08-f3aa-4799-9cba-7e551ed8b00a) | ![fold](https://github.com/user-attachments/assets/31f93f6d-6d0f-440b-b8d8-504447fdb330) |

## Checklist
 
#### I have tested any UI changes...
